File: test/fixtures/javascript/get_with_data2.js
fetch('http://localhost:28139/v2/alerts_policy_channels.json?policy_id=policy_id&channel_ids=channel_id', {
  method: 'PUT',
  headers: {
    'X-Api-Key': '{admin_api_key}',
    'Content-Type': 'application/json'
  }
});

File: test/fixtures/kotlin/get_with_data2.kt
import java.io.IOException
import okhttp3.OkHttpClient
import okhttp3.Request
import okhttp3.RequestBody.Companion.toRequestBody

val client = OkHttpClient()

val request = Request.Builder()
  .url("http://localhost:28139/v2/alerts_policy_channels.json?policy_id=policy_id&channel_ids=channel_id")
  .method("PUT", "".toRequestBody())
  .header("X-Api-Key", "{admin_api_key}")
  .header("Content-Type", "application/json")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}
